microsoft-edge/web-platform/user-agent-guidance.md

```diff
@@ -1,70 +1,88 @@
-    
-When [feature detection][MdnLearnToolsTestingCrossBrowserTestingFeatureDetection] isn't applicable and you must use user agent detection, use the following format to detect Microsoft Edge user agent on Windows and Android.  
+User-Agent Client Hints is a cleaner, more privacy-preserving way to access browser information such as browser name, version number, platform, and more. Soon the User-Agent string will be frozen and deprecated by various browsers. For example, the Chrome Platform Status site describes the change in [Feature: Reduce User-Agent string information][https://www.chromestatus.com/feature/5704553745874944].
 
-## User-Agent Client Hints  
+Employ User-Agent Client Hints when you want to:
+- Determine if new browser activity is from the expected user.
+- Customize hints or instructions if the user is new to this site.
 
-Starting in Microsoft Edge version 90, access browser information in a cleaner, more privacy-preserving way.  
+Do not use User-Agent Client Hints to:
+- Block *unsupported* browsers.
+- Restrict access to features on your site.
 
-### User-Agent Client Hints HTTPS Header  
+For more information, navigate to the at [W3C Community Draft Report | User-Agent Client Hints][https://wicg.github.io/ua-client-hints/].
 
-By default, Chromium browsers including Microsoft Edge send the `Accept-CH-UA` response header in the following format.  
+### User-Agent Client Hints HTTPS header
+
+When Microsoft Edge sends an HTTPS request to a server, it sends the User-Agent Client Hints headers that correspond to browser brand and mobile information. If the server requires more granular information about the browser, its response includes an `Accept-CH` header. The value of that response header is a comma-separated list of all the Client Hints request headers the server wants from the browser, such as `Accept-CH: Sec-CH-UA-Platform,Sec-CH-UA-Platform-Version`. The next Microsoft Edge HTTPS request to the server will include the requested User-Agent Client Hints headers.
+
+By default, Chromium browsers including Microsoft Edge send the `Sec-CH-UA` and `Sec-CH-UA-Mobile` response header in the following format.  
 
 ```https
-Sec-CH-UA: "Chromium";v="91", "Microsoft Edge";v="91","Dummy;Browser Brand";v="99"
+Sec-CH-UA: "Chromium";v="92", "Microsoft Edge";v="92","Placeholder;Browser Brand";v="99"
 Sec-CH-UA-Mobile: ?0
 ```  
 
-> [!NOTE]
-> Client Hints are only sent over secure connections using `HTTPS`.  
-
-The following low entropy hints are sent by default.  If you need to access more information, send one of the following request headers.  
-
-#### User-Agent request and response headers  
+The following table shows all available hints request headers with sample values.
 
-| Request header | Example response value |  
+| User-Agent Request header | Example User-Agent response value |  
 |:--- |:--- |  
 | `Sec-CH-UA` | `"Chromium";v="91", "Microsoft Edge";v="91","GREASE";v="99"` |  
 | `Sec-CH-UA-Mobile` | `false` |  

```

Comment:
In UA client hints headers, `false` is represented as `?0`. Could we update this?

Reply:
Added ?0 and ?1 as examples.

Reply:
In each other entry in the table, though, there is only one example. Could we only have one here too? Since we include `?0` above, maybe `?1` would be a good example here